react-refresher/my-app/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import Counter from './Counter';
+
+function App() {
+  return (
+    <>
+      <Counter />
+    </>
```

Comment:
No fragments needed here since we're only rendering 1 thing. We only need to include fragments if we're running multiple sibling elements

Reply:
Oh yeah, that's right! I'll make a note of that for the future.